src/bin/rm/rm.rs

```diff
@@ -15,13 +15,34 @@ pub struct RmArgs {
     crates: Vec<String>,
 
     /// Remove crate as development dependency.
-    #[clap(long, short = 'D', conflicts_with = "build")]
+    #[clap(
+        long,
+        short = 'D',
+        conflicts_with = "build",
+        help_heading = "SECTION",
+        group = "SECTION"
+    )]
     dev: bool,
 
     /// Remove crate as build dependency.
-    #[clap(long, short = 'B', conflicts_with = "dev")]
+    #[clap(
+        long,
+        short = 'B',
+        conflicts_with = "dev",
+        help_heading = "SECTION",
+        group = "SECTION"
+    )]
     build: bool,
 
+    /// Remove as dependency from the given target platform.
+    #[clap(
+        long,
+        forbid_empty_values = true,
+        help_heading = "SECTION",
+        group = "section"
```

Comment:
How come `group` is still being set?  Its a different case, so its a different group but probably should remove it if its not being used.

Reply:
I haven't made any changes here since your last comment. Not sure whether you wanted all instances of group removed, or just on `target`. I assume the latter?

Reply:
Let's start simple and remove them all.

We should slowly move to be more like `cargo add` in structure which would mean replacing `conflicts_with` by setting `group = "section"`.

Reply:
Okay, that's what I figured. Thanks!